docs/problem_formulation/problem_formulation_ocp_mex.tex

```diff
@@ -156,13 +168,12 @@ \section{Problem Formulation}\label{sec:problem}
 
 %
 \paragraph{Document Purpose.}
-This document describes the \matlab{} interface of \acados.
-Here, the focus is to give a mathematical overview of the problem formulation and possible options to model it within \acados.
-The problem formulation and the possibilities of \acados{} are similar in the \python{} interface, however, some of the string identifiers are different.
-The documentation is not exhaustive and does not contain a full description for the \matlab{} interface.
+This document describes the class of optimal control problems that can be formulated and solved in \acados{} via the class \texttt{AcadosOcp}, available in the \matlab{} and \python{} interfaces.
+%Here, the focus is to give a mathematical overview of the problem formulation and possible options to model it within \acados.
+This documentation is (most likely) not exhaustive and does not contain a full description of all options provided by the \python{} and \matlab{} interfaces.
 
-You can find examples in the directory \code{<acados>/examples/acados\_matlab\_octave}.
-The source code of the \acados{} \matlab{} interface is found in: \code{<acados>/interfaces/acados\_matlab\_octave} and should serve as a more extensive, complete and up-to-date documentation about the possibilities.
+%You can find examples in the directory \code{<acados>/examples/acados\_matlab\_octave}.
+%The source code of the \acados{} \matlab{} interface is found in: \code{<acados>/interfaces/acados\_matlab\_octave} and should serve as a more extensive, complete and up-to-date documentation about the possibilities.
```

Comment:
The following paragraph "Abbreviations" seems to be not properly used anymore.
This paragraph is mainly to explain `Z*` penalties and `J*` matrices.

The `Z*` slack penalties are after the overhaul given as vectors, before diagonal matrices.
There are two things that should be clarified in this PDF: the restriction (not any matrix) and the way they are to be set.

I suggest adding a remark for `J*`/ `Js*` matrices: explaining restrictions on what they can be and how they relate to `idx*`. Then link to this remark in section 1 and the constraint section.

I would suggest to handle the `Z*` similarly.
Does that make sense?

Reply:
I shortened this section a bit and provide an example. Maybe check this one again :see_no_evil: